packages/twenty-front/src/pages/settings/data-model/SettingsObjectDetailPage.tsx

```diff
@@ -167,7 +181,31 @@ export const SettingsObjectDetailPage = () => {
             children: t`Workspace`,
             href: getSettingsPath(SettingsPath.Workspace),
           },
-          { children: t`Objects`, href: getSettingsPath(SettingsPath.Objects) },
+          ...(isDefined(applicationName)
```

Comment:
Same I don't think we should do this. What about fields declared by application then? Company will most likely have several fields by several applications